### PR TITLE
Couple of fixups

### PR DESCRIPTION
**Pull request by surminus**

### CODE REVIEW

File: projects/analytics_lambdas/resources/main.tf [hunk 1]
```diff
@@ -31,7 +31,7 @@ data "aws_iam_policy_document" "s3-access-ro" {
 
 resource "aws_iam_policy" "s3-access-ro" {
   name = "s3-access-ro"
-  path = "${var.public_api_logs_prefix}"
+  path = "/"
```
Comment (deanwilson) [inline projects/analytics_lambdas/resources/main.tf]: What's the reason behind this change?

**Comment by surminus** [comment]
This never worked in the end. I did this while pairing with @steventux, and don't think it was actually ever used for deployment of the lambda?

**Comment by steventux** [comment]
The reason behind the change is that the path `public_api_logs/` threw errors when we attempted to deploy this. I understand this to be the path the lambda can access in the bucket.
I think this change is valid, but moot. We couldn't use the deployed lambda resulting from this config because of permissions problems.
When we attempted to test it the lambda didn't have sufficient permissions to read from the bucket.
It's since been deployed using the aws console, using terraform to deploy it would be the preferred way but given the time constraints we didn't get this working.

**Comment by surminus** [comment]
I'm just going to close this, then.
